recipes/glückskekse.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Fortune cookies"
+image: https://www.smarticular.net/wp-content/uploads/2019/12/glueckskekse-selber-machen-backen-rezept-1-fb.jpg
```

Comment:
Please add your author information, for example:

```
author: Marie Bogena
```

or, alternatively, your GitHub handle.